builtin/providers/cloudstack/resource_cloudstack_instance.go

```diff
@@ -82,6 +82,11 @@ func resourceCloudStackInstance() *schema.Resource {
 				Optional: true,
 				Default:  false,
 			},
+			"project_name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  nil,
```

Comment:
Is it possible to move a VM from one project to another? If so the code to do so should be added in the update function of this resource. If not (what I suspect, but I didn't check it) we should add `ForceNew: true` here indicating that when you change the project for this resource, it needs to recreated.